tests/vmi_lifecycle_test.go

```diff
@@ -571,6 +573,69 @@ var _ = Describe("[rfe_id:273][crit:high][vendor:cnv-qe@redhat.com][level:compon
 
 				tests.WaitForSuccessfulVMIStart(newVMI)
 			})
+
+			It("[test_id:5275]VMIs with Bridge Networking shouldn't fail after the kubelet restarts", func() {
+				if currentConfiguration.NetworkConfiguration == nil {
+					currentConfiguration.NetworkConfiguration = &v1.NetworkConfiguration{}
+				}
+
+				currentConfiguration.NetworkConfiguration.PermitBridgeInterfaceOnPodNetwork = pointer.BoolPtr(true)
+				kv := tests.UpdateKubeVirtConfigValueAndWait(currentConfiguration)
+				currentConfiguration = kv.Spec.Configuration
+
+				By("starting a VMI on a node")
+				vmi, err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+				Expect(err).To(BeNil(), "Should submit VMI successfully")
+
+				// Start a VirtualMachineInstance with bridged networking
+				nodeName := tests.WaitForSuccessfulVMIStart(vmi)
+
+				By("triggering a device plugin re-registration on that node")
+				pod, err := kubecli.NewVirtHandlerClient(virtClient).Namespace(flags.KubeVirtInstallNamespace).ForNode(nodeName).Pod()
+				Expect(err).ToNot(HaveOccurred())
+
+				_, _, err = tests.ExecuteCommandOnPodV2(virtClient, pod,
+					"virt-handler",
+					[]string{
+						"rm",
+						// We want to fail if the file does not exist, but don't want to be asked
+						// if we really want to remove write-protected files
+						"--interactive=never",
+						device_manager.SocketPath("kvm"),
+					})
+				Expect(err).ToNot(HaveOccurred())
```

Comment:
does this actually cause a kubelet restart? or are we simulating what would happen if a kubelet actually restarted? 

I guess i don't understand why we don't actually kill the kubelet on the node and explicitly force a restart of the kubelet.

Reply:
This is a simulation.  I wasn't sure how far I should take it, so I went with the conservative approach first.  Let's see what I can come up with for an actual restart on the node.

Reply:
I'm not sure how to test locally, so hopefully the gate will give me some insight into the right direction.